test/clojure/core_test/nil_questionmark.cljc

```diff
@@ -0,0 +1,16 @@
+(ns clojure.core-test.nil-questionmark
+  (:require [clojure.test :as t]))
+
+(t/deftest common
+  (t/are [in ex] (= (nil? in) ex)
+    nil true
+    0 false
+    "" false
+    "nil" false
+    :nil false
+    ##Inf false
+    ##NaN false
+    0.0 false
+    {} false
+    '() false
+    [] false))
```

Comment:
It'd be good to add an infinite sequence test, á la: https://github.com/jank-lang/clojure.core-test/blob/main/test/clojure/core_test/any.cljc#L13

Reply:
Done. I can add it in a separate `deftest` if you prefer. I also added a test for `false` because I realized I missed it the first time around.

Reply:
Nice! I think it's good to be explicit about it in a separate test. `common` is meant for normal cases, while other edge cases have their own tests.

Reply:
Ok - changed!

Reply:
:muscle: Thanks for helping!